frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -361,14 +392,35 @@ function ReaderHighlight:addToMainMenu(menu_items)
     end
     menu_items.translation_settings = Translator:genSettingsMenu()
 
-    if not Device:isTouchDevice() then
-        -- Menu items below aren't needed.
-        return
-    end
-
     menu_items.long_press = {
         text = _("Long-press on text"),
         sub_item_table = {
+            {
+                text = _("Highlight long-press interval"),
+                keep_menu_open = true,
+                callback = function()
+                    local SpinWidget = require("ui/widget/spinwidget")
+                    local items = SpinWidget:new{
+                        title_text = _("Highlight long-press interval"),
+                        info_text = _([[
+If a touch is not released in this interval, it is considered a long-press. On document text, single word selection will not be triggered.
+
+The interval value is in seconds and can range from 3 seconds to 20 seconds.]]),
```

Comment:
> It is copy from here and small modifications: https://github.com/koreader/koreader/blob/23d12c3f04a2d1135c0eb76f4678b7017a78f7ec/plugins/gestures.koplugin/main.lua#L574

Thanks, that's what I needed to know.

I'd suggest to remove that first seconds because it's a bit awkward here, but to otherwise keep it the same as the other text.
```suggestion
If a touch is not released in this interval, it is considered a long-press. On document text, single word selection will not be triggered.

The interval value is in seconds and can range from 3 to 20 seconds.]]),
```